src/devices/bus/multibus/multibus.h

```diff
@@ -87,6 +87,12 @@ class multibus_device
 	template <unsigned I> auto int_callback() { return m_int_cb[I].bind(); }
 	template <unsigned I> void int_w(int state) { m_int_cb[I](state); }
 
+	// wait signal (split for read/write operations)
+	auto wait_io_rd_cb() { return m_wait_rd_cb.bind(); }
+	auto wait_io_wr_cb() { return m_wait_wr_cb.bind(); }
+	void set_wait_io_rd(int state) { m_wait_rd_cb(state); }
+	void set_wait_io_wr(int state) { m_wait_wr_cb(state); }
```

Comment:
You’re mixing live signal methods with configuration methods.  This leads to confusion – they should be clearly separated.